isisd/isis_tlvs.c

```diff
@@ -3593,6 +4527,23 @@ void isis_tlvs_set_dynamic_hostname(struct isis_tlvs *tlvs,
 		tlvs->hostname = XSTRDUP(MTYPE_ISIS_TLV, hostname);
 }
 
+/* Set Router Capability TLV parameters */
+void isis_tlvs_set_router_capability(struct isis_tlvs *tlvs,
+				     struct in_addr router_id,
+				     const struct isis_router_cap *cap)
```

Comment:
Isn't `router_id` redundant now that the third parameter is a full `isis_router_cap` TLV?

Reply:
done